build.rs

```diff
@@ -0,0 +1,17 @@
+use cfg_aliases::cfg_aliases;
+
+fn main() {
+    // Add alias to see if any symphonia features are enabled
+    // This prevents having to copy/paste this large cfg check each time
+    cfg_aliases! {
```

Comment:
Is this build.rs file needed at all? I've thought enabling `symphonia-mp3` feature for example enables the `symphonia` feature anyways.

Reply:
Oh, you're right. I didn't realize `#[cfg(feature = "symphonia")]` would work since there isn't an explicit feature just called symphonia. I'll get rid of this and use that instead.